editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
```

Comment:
I think matching the slug for the block would be a good place to start.  That way the css classes are namespaced like the blocks are.